sql/2025/page-weight/good-ni-poor-percent-cwv-by-mb.sql

```diff
@@ -0,0 +1,339 @@
+WITH base_data AS (
+  SELECT
+    date,
+    client,
+    is_root_page,
+    CAST(JSON_VALUE(summary.bytesTotal) AS INT64) AS bytes_total,
+    -- these are page level CrUX metrics, pages may not have all / any metrics available, but do represent the user experience of the measured page weight vs. origin.
+    CAST(JSON_VALUE(summary.crux.metrics.largest_contentful_paint.percentiles.p75) AS FLOAT64) / 1000 AS lcp,
+    CAST(JSON_VALUE(summary.crux.metrics.cumulative_layout_shift.percentiles.p75) AS FLOAT64) AS cls,
+    CAST(JSON_VALUE(summary.crux.metrics.interaction_to_next_paint.percentiles.p75) AS FLOAT64) AS inp,
+    CAST(JSON_VALUE(summary.crux.metrics.first_contentful_paint.percentiles.p75) AS FLOAT64) / 1000 AS fcp,
+    CAST(JSON_VALUE(summary.crux.metrics.experimental_time_to_first_byte.percentiles.p75) AS FLOAT64) AS ttfb
+  FROM
+    `httparchive.crawl.pages`
+  WHERE
+    date = '2025-07-01'
+),
+
+lcp_data AS (
+  SELECT *
+  FROM base_data
+  WHERE lcp IS NOT NULL
+),
```

Comment:
Do you need all these `XXX_data` tables given you're about to have `XXX_metrics` tables. Can't you just move the `WHERE xxx IS NOT NULL` clause into those metric tables?